packages/client-core/src/user/components/UserMenu/menus/ProfileMenu.tsx

```diff
@@ -463,7 +464,7 @@ const ProfileMenu = ({ hideLogin, onClose, isPopover }: Props): JSX.Element => {
               </Text>
             )}
 
-            {!originallyAcceptedTOS.value && (
+            {isGuest && (
```

Comment:
The TOS acceptance and 13orOver checkbox will never be saved for guest users since it's only going off of isGuest. Is that what we want? Or should a guest user have this information saved so they don't have to re-check the boxes every time they go to a location?

Reply:
The idea here is that log in is not allowed until they accept the TOS & verify 13+, the only information that is saved is 18+, which is saved as the accountTOS field in the user table, which i suppose we should rename, but i'm ok with it as is for now until we have the new UI & flow implemented.

Reply:
Guests do not have any restrictions, the restriction is on media/chat, which is restricted to non-guest users who have checked 18+